### PR TITLE
Moved JsonCheck to ApiChecker.Check namespace

### PR DESCRIPTION
**Pull request by elbow-jason**
Tests are passing.

### CODE REVIEW

**Comment by elbow-jason** [comment]
@sabondano Gooooooooo!!!

File: test/api_checker/check/json_check/jsonapi_test.exs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+defmodule ApiChecker.Checker.JsonCheck.JsonapiTest do
```
Comment (sabondano) [inline test/api_checker/check/json_check/jsonapi_test.exs]: typo? `Checker` -> `Check`

File: test/api_checker/periodic_task_test.exs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 defmodule ApiChecker.PeriodicTaskTest do
   use ExUnit.Case, async: true
   alias ApiChecker.{PeriodicTask, JsonCheck}
```
Comment (sabondano) [inline test/api_checker/periodic_task_test.exs]: remove `JsonCheck`?

File: test/api_checker/check/json_check/jsonapi_test.exs (hunk 1 above)
Comment (elbow-jason) [reply test/api_checker/check/json_check/jsonapi_test.exs]: fixed.

File: test/api_checker/periodic_task_test.exs (hunk 2 above)
Comment (elbow-jason) [reply test/api_checker/periodic_task_test.exs]: and fixed

**Comment by elbow-jason** [comment]
🍰 🍰 🍰 🍰 🍰 🍰 🍰 🍰 🍰 🍰 🍰 🍰 🍰

**Pull request merged**